### DISCUSSION TITLE
I see that this feature was merged to master. Is the PAR feature planned to integrate into the ory/hydra project?

### DISCUSSION THREAD

**Discussion by justayar** [ory/hydra#3218] [Q&A] [answered]
I see that this feature was merged to master. Is the PAR feature planned to integrate into the ory/hydra project?

_Originally posted by @justayar in https://github.com/ory/hydra/discussions/3071#discussioncomment-3222091_

**Comment by vinckr** [comment]
It will be included in the next release @justayar - coming soon!

**Comment by justayar** [comment]
Do you mean Hydra v2.0? @vinckr

**Comment by vinckr** [comment]
Yes, watch this PR: https://github.com/ory/hydra/pull/2796

**Comment by aeneasr** [comment, marked as answer]
Sorry to be the bearer of bad news, but PAR is not automatically included in Ory Hydra 2.0 as it needs work. I've answered this before here: https://github.com/ory/fosite/pull/660#issuecomment-1188958354
